packages/transforms/mocking/yaml-config.graphql

```diff
@@ -0,0 +1,16 @@
+extend type Transform {
+    mocking: MockingConfig
+}
+
+type MockingConfig {
+    if: Boolean
+    preserveResolvers: Boolean
+    mocks: [MockingFieldConfig]
+}
+
+type MockingFieldConfig {
+    apply: String!
+    if: Boolean
```

Comment:
I think this one should be `String` because it represents the value from the YAML, before interpolation, which could be also something like `${MY_VAR}`

Reply:
Or, a scalar that represents `String | Boolean`

Reply:
Already done but not pushed yet :D